Serializer/Jms/Handler/ResourceHandler.php

```diff
@@ -85,6 +86,7 @@ private function doSerializeResource(Resource $resource, $depth = 0)
         }
 
         $data['path'] = $resource->getPath();
+        $data['node_name'] = PathHelper::getNodeName($data['path']);
```

Comment:
Yeah, I think label makes sense. Although we should probably change it to `resource_label`, to not conflict with a property called label? (shouldn't we add properties in their own list under `properties`?)

The Tree uses this node_name by default.